src/m_menu.c

```diff
@@ -3924,8 +3935,6 @@ setup_menu_t gen_settings1[] = { // General Settings screen1
   {"Vertical Sync", S_YESNO, m_null, M_X,
    M_Y+ gen1_vsync*M_SPC, {"use_vsync"}, 0, I_ResetScreen},
 
-  {"", S_SKIP, m_null, M_X, M_Y + gen1_stub1*M_SPC},
```

Comment:
I'm a bit sad for this gap. :cry:

Reply:
Maybe we can move the translucency to "Display Options", but we don't have much space here either. Also we can cut some options, for example we can move "Sound Channels" to config only. Perhaps it's time to rearrange menus again 😄